lib/util.js

```diff
@@ -13,6 +13,8 @@ exports.format = function(f) {
     return objects.join(' ');
   }
 
+  if (arguments.length === 1) return f;
+
   var i = 1;
   var args = arguments;
```

Comment:
It would probably also help to __not__ leak the arguments here.

Maybe consider https://gist.github.com/Fishrock123/98c35a0c745cb59d7496